tests/misc_env/push_cosbench_workload.py

```diff
@@ -160,9 +160,33 @@ def push_workload(controller, client, workload_file_name):
     LOG.info(out)
     wid = out.strip().split(": ")[1]
 
+    sleep_time = 30
```

Comment:
Shall we take sleep_time and retry_limit as an argument to the method?

Reply:
As discussed, we are keeping below approach in mind for the future.
we can default that value to whatever we have now, keep a way we can fetch it from config, in case if we want different retry/sleep time based on workload

for now we can go ahead with current approach!